GOVERNANCE.md

```diff
@@ -0,0 +1,159 @@
+
+## Voting
+
+The Prometheus Project usually runs by informal consensus, however sometimes a formal decision must be made.
+
+Depending on the subject matter, as laid out in [Decisionmaking](#decisionmaking), different methods of voting are used.
+
+For all votes, voting must be open for at least one week. A vote may be called and closed early if enough votes have come in one way that further votes cannot change the final decision.
+
+In all cases, all and only [team members](#team-members) are eligible to vote, with the sole exception of the forced removal of a team member, in which said member is not eligible to vote.
+
+Discussion and votes on personal matters (including but not limited to team membership and maintainership) are held in private on the [team mailing list][team]. All other discussion and votes are held in public on the [developer mailing list][team].
+
+For public discussions, anyone interested is encouraged to participate. Formal power to object or vote is limited to [team members](#team-members).
+
+### Consensus
+
+The default decision making mechanism for the Prometheus Project is [lazy consensus][lazy]. This means that any decision on technical issues is considered supported by the [team][team] as long as nobody objects.
+
+Silence on any consensus decision is implicit agreement and equivalent to explicit agreement. Explicit agreement may be stated at will. Decisions may, but do not need to be called out and put up for decision on the [developers mailing list][devs] at any time and by anyone.
+
+Consensus decisions can never override or go against the spirit of an earlier explicit vote.
+
+If any [team member](#team-members) raises objections, we work together towards a solution that all involved can accept. This solution is again subject to lazy consensus.
+
+In case no consensus can be found, but a decision one way or the other must be made, any [team member](#team-members) may call a formal [majority vote](#majority-vote).
+
+### Majority vote
+
+Majority votes must be called explicitly in a separate thread on the appropriate mailing list. The subject must be prefixed with `[VOTE]`. In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+
+Voting must be open for at least 1 week (7 days from the beginning of the vote). The end date should be clearly stated in the call to vote.
+
+Votes may take the form of a single proposal, with the option to vote yes or no; or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if more vote in favour than against.
+
+A vote on multiple alternatives is considered decided in favour of one alternative if any alternative receives more than half of the total votes cast. Members may vote against all alternatives by voting "no". Should no alternative reach a majority, another vote on a reduced number of options may be called separately.
```

Comment:
I expanded on the "multiple choice" type vote for clarity. I did this so that we a) don't *have* to formulate every question as a yes/no and b) nothing can be decided that does not have a majority.

It is up to anyone who wants to get to a final decision to take a failed multi-way vote and distill either another multi-way or a yes/no vote from it (or don't).

Reply:
How about integrating approval voting into this, so if I like two out of three choices, I can vote for both of them? I think voting "no" can still be an option then, and the rest of your rules would still work? So:

- multiple options + "no" are available for voting
- you can approve of any number of them, except that "no" must be exclusive
- whichever alternative receives the most approvals from more than half of the people who voted wins

Or is this too complicated?

Reply:
No, I think that's a good way to make multiple-alternative votes go quicker and cleaner.